src/vm/i386/jitinterfacex86.cpp

```diff
@@ -34,6 +34,39 @@
 #define MON_DEBUG 1
 #endif
 
+#ifdef _TARGET_X86_
```

Comment:
I do not think this ifdef is needed here.  This file is only compiled for x86. You can fix it together with the bug that you are looking at.